libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -13,6 +13,7 @@ import {
 } from '../utilities.js';
 import { decorateLinks } from '../../../../utils/utils.js';
 import { replaceText } from '../../../../features/placeholders.js';
+import { CONFIG } from '../../global-navigation.js';
```

Comment:
Why add the icon to the main file, when it can be added here directly? Maybe a navigation doesn't include dropdown menus, such as the plans pages. Then you have that icon with no use in the main file. Are there other reasons for it being there?